snapcraft/tests/unit/store/test_store_client.py

```diff
@@ -1506,3 +1517,13 @@ def test_conflicting_force(self):
             # force the update, even on conflicts!
             result = self.client.push_binary_metadata('basic', metadata, True)
         self.assertIsNone(result)
+
+    def test_snap_not_found(self):
```

Comment:
Please rename this test.

Reply:
@elopio What should I change it to?

Reply:
Maybe call the first one test_snap_not_found_in_push_metadata, and this one test_snap_not_found_in_binary_metadata.

An alternative would be to use testscenarios. Your two tests are the same, so you can move them to their own test case, and pass the function to call in a scenario.
https://pypi.python.org/pypi/testscenarios

Reply:
Thanks for letting me know about testscenarios, this looks better because I also hate repeated code.